Containerfile

```diff
@@ -0,0 +1,126 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# Copyright (C) 2023 Olliver Schinagl <oliver@schinagl.nl>
+
+ARG ALPINE_VERSION="latest"
+ARG TARGET_ARCH="library"
+
+FROM docker.io/${TARGET_ARCH}/alpine:${ALPINE_VERSION} AS builder
+
+WORKDIR /src
+
+COPY . /src/
+
+RUN apk add --no-cache \
+        'avahi-dev' \
+        'bash' \
+        'bsd-compat-headers' \
+        'build-base' \
+        'cmake' \
+        'coreutils' \
+        'dbus-dev' \
+        'ffmpeg4-dev' \
+        'findutils' \
+        'gettext-dev' \
+        'git' \
+        'gnu-libiconv-dev' \
+        'libdvbcsa-dev' \
+        'libhdhomerun-dev' \
+        'libva-dev' \
+        'libvpx-dev' \
+        'linux-headers' \
+        'musl-dev' \
+        'openssl-dev>3' \
+        'opus-dev' \
+        'pngquant' \
+        'python3' \
+        'uriparser-dev' \
+        'wget' \
+        'x264-dev' \
+        'x265-dev' \
+        'zlib-dev' \
+    && \
+    git config --global --add safe.directory '/src' && \
+    ./configure \
+                --prefix='/usr/local' \
+                --mandir='/doc/man' \
+                --disable-execinfo \
+                --disable-ffmpeg_static \
+                --disable-hdhomerun_static \
+                --disable-libfdkaac_static \
+                --disable-libmfx_static \
+                --disable-libopus_static \
+                --disable-libtheora_static \
+                --disable-libvorbis_static \
+                --disable-libvpx_static \
+                --disable-libx264_static \
+                --disable-libx265_static \
+                --enable-bundle \
+                --enable-dvbcsa \
+                --enable-hdhomerun_client \
+                --enable-kqueue \
+                --enable-libav \
+                --enable-nvenc \
+                --enable-pngquant \
+                --enable-qsv \
+                --nowerror \
+                --python=python3 \
+        && \
+        make DESTDIR='/tvheadend' -j$(($(nproc) - 1)) install && \
+        rm -f -r '/tvheadend/doc'
+
+FROM docker.io/${TARGET_ARCH}/alpine:${ALPINE_VERSION}
+
+LABEL maintainer="Olliver Schinagl <oliver@schinagl.nl>"
+
+EXPOSE 9981 \
```

Comment:
:9983 for sat>ip default port + :554

Reply:
I'm trying to figure out what this means. I see that the cmdline states that
```
      --satip_rtsp            SAT>IP RTSP port number for server
                              (default: -1 = disable, 0 = webconfig, standard port is 554)
```
So Default is no port, ok fair, 0 means that it uses the same port as webconfig, which is 9981. But then what is 'standard port'? Who/what is using 554? When is it binding to 554? if you do `--satip_rtsp 554`? Yes, the 'standard' rtsp port is 554, but nothing is using it.

Since 'expose' is really just for documentation purposes of the container, and by 'default' we don't listen on this port, I think it's best to leave it out.

Further more
```
### 1. Define the RTSP Port

This can be anything you like, it is recommended that you use 9983 
(to avoid permission issues). Entering zero (0) in this field will 
disable the server. 
```
So this basically recommends you to do `--satip_rtsp 9983` yeah? Which means the container by default doesn't use 9983, so we shouldn't expose it.

Granted, both notes should be put in the documentation :) (I'm trying to find a place to put it though).